### PR TITLE
Update custom-resource-application.md

### PR DESCRIPTION
**Pull request by dexhorthy**

### CODE REVIEW

**Review by paigecalvert — DISMISSED** [review]
@dexhorthy, pushed some edits--take a look and feel free to merge if it looks good to you!

File: docs/reference/custom-resource-application.md [hunk 1]
```diff
@@ -114,7 +114,11 @@ This option is applicable to existing clusters only.
...
 ## ports
-These are extra ports (additional to the :8800 admin console port) that should be port-forwarded when running the `kots admin-console` command.
+These are extra ports (additional to the :8800 admin console port) that are port-forwarded when running the `kots admin-console` command.
+
+You can use the `ports` field to create a port-forward to a service that has a `ClusterIP` type. For clusters provisioned by the Kubernetes installer, you can also create a custom link to a service that has a `NodePort` type.
+
+For more information about configuring a custom link in Kubernetes installer clusters to a `NodePort` type service, see [Adding Buttons and Links](/vendor/admin-console-adding-buttons-links).
 
 ### serviceName
 The name of the service that has a `ClusterIP` type that should receive the traffic.
```
Comment (jdewinne) [inline docs/reference/custom-resource-application.md:124]:
```suggestion
The name of the service that has a `ClusterIP` type or `NodePort` type if using the Kubernetes installer, that should receive the traffic.
```

**Review by jdewinne — CHANGES_REQUESTED** [review]
I would also change the `serviceName` section to make it clear when `NodePort` can be used.

File: docs/reference/custom-resource-application.md (hunk 1 above)
Comment (paigecalvert) [reply docs/reference/custom-resource-application.md:124]: Nice 👍

**Comment by paigecalvert** [comment]
@jdewinne Great suggestion--I made the change. Looks like this is blocked on your approval since you used the "changes requested" feature. Would you be able to take a look (and merge) if this looks ready to go?

**Review by jdewinne — APPROVED** [review]

**Comment by jdewinne** [comment]
> @jdewinne Great suggestion--I made the change. Looks like this is blocked on your approval since you used the "changes requested" feature. Would you be able to take a look (and merge) if this looks ready to go?

@paigecalvert lgtm!

**Comment by paigecalvert** [comment]
Sweet thanks @jdewinne !

**Pull request merged**